models/classes/runner/session/TestSession.php

```diff
@@ -150,28 +150,14 @@ public function setTimerTarget($timerTarget)
      */
     public function getItemTags(RouteItem $routeItem)
     {
-        $test = $routeItem->getAssessmentTest();
-        $testPart = $routeItem->getTestPart();
-        $sections = $routeItem->getAssessmentSections();
-        $sections->rewind();
-        $sectionId = key(current($sections));
         $itemRef = $routeItem->getAssessmentItemRef();
         $itemId = $itemRef->getIdentifier();
         $occurrence = $routeItem->getOccurence();
 
-        $tags = [
-            $itemId,
+        return [
             $itemId . '#' . $occurrence,
-            $sectionId,
-            $testPart->getIdentifier(),
-            $test->getIdentifier(),
-            $itemRef->getHref(),
         ];
 
-        if ($this->isRunning() === true) {
-            $tags[] = $this->getItemAttemptTag($routeItem);
```

Comment:
I'd keep this one too, otherwise you won't be able to differentiate different attempts o

Reply:
I agree and I restore it, however this tag is the most heavy.